appraisal.lic

```diff
@@ -10,25 +10,35 @@ class Appraisal
 
   def initialize
     @equipment_manager = EquipmentManager.new
-    @equipment_manager.wear_equipment_set?('standard')
     @settings = get_settings
+    train_list = get_settings.appraisal_training
+    
+    train_list.each do |task| 
+      break if DRSkill.getxp('Appraisal') >= 30
+      case task
+      when /zill/
```

Comment:
I had the idea that the user might enter something like `gems` instead of `pouch` or `pouches`, but it seems half-baked and adds more complexity.. ill ditch the multiple options. I do like the regex though, since it lets the user write `pouch`, `pouches`, `zill`, `zills`. Is it something you feel strongly about? I don't feel strongly about it, but i do like that it covers the user unintentionally entering the plural vs singular or vice versa.